scripts/settings.js

```diff
@@ -38,10 +37,10 @@ function getDefaultCurrency() {
     return getCurrencyByAlpha2(langCode)?.toLowerCase() || 'usd';
 }
 /**
- * The global market data source
- * @type {CoinGecko}
+ * The user-selected Price Oracle, used for all price data
+ * @type {Oracle}
  */
-export let cMarket = new CoinGecko();
+export let cOracle = new Oracle();
```

Comment:
why is the cOracle created here and not in the price.js file? I'd suggest moving it to price.js

Reply:
In the future we can add the option for multiple Oracles in MPW (just like we have multiple Blockbooks and RPCs) so I think keeping it in the Settings follows our usual settings-based pattern, I left a TODO in the Oracle class JSDoc regarding this as well.